frontend/device/kobo/powerd.lua

```diff
@@ -2,13 +2,19 @@ local BasePowerD = require("device/generic/powerd")
 local NickelConf = require("device/kobo/nickel_conf")
 
 local KoboPowerD = BasePowerD:new{
-    fl_min = 0, fl_max = 100,
+    -- Do not actively set front light to 0, it may confuse users -- clicking
+    -- hardware button won't take any effect.
+    fl_min = 1, fl_max = 100,
     flIntensity = 20,
     restore_settings = true,
     fl = nil,
 
-    batt_capacity_file = "/sys/devices/platform/pmic_battery.1/power_supply/mc13892_bat/capacity",
-    is_charging_file = "/sys/devices/platform/pmic_battery.1/power_supply/mc13892_bat/status",
+    flState = false,
+    batt_capacity_file =
+        "/sys/devices/platform/pmic_battery.1/" ..
```

Comment:
If you want to wrap the line, it's more readable to define a local var `/sys/devices/platform/pmic_battery.1/power_supply/mc13892_bat/` at the top of the file and reuse it here ;)